packages/browser-wallet/src/popup/shared/Form/Form.tsx

```diff
@@ -23,7 +23,7 @@ export const useForm = <TFormValues extends FieldValues = FieldValues>(
     props?: UseFormProps<TFormValues>
 ): UseFormReturn<TFormValues> => useFormLib<TFormValues>({ ...useFormDefaults, ...props });
 
-type FormProps<TFormValues> = ClassName & {
+type FormProps<TFormValues extends FieldValues = FieldValues> = ClassName & {
```

Comment:
:thinking:

Reply:
Again, upgrading typescript triggered this.